deployment/update_production.sh

```diff
@@ -14,20 +18,17 @@ if [ $# -eq 1 ] # if there is exactly one argument
         set -x # print executed commands
 fi
 
-sudo -H -u evap git fetch
 sudo -H -u evap git checkout origin/release
 sudo -H -u evap pip3 install --user -r requirements.txt
 sudo -H -u evap ./manage.py compilemessages
 sudo -H -u evap ./manage.py collectstatic --noinput
 sudo -H -u evap ./manage.py compress --verbosity=0
 sudo -H -u evap ./manage.py migrate
-# reload only after static files are updated, so the new code finds all the files it expects.
-# also, reload after migrations happened. see https://github.com/fsr-itse/EvaP/pull/817 for a discussion.
-sudo service apache2 reload
-# update caches. this can take minutes but doesn't need a reload.
 sudo -H -u evap ./manage.py clear_cache
```

Comment:
Maybe link the issue so we will never fall for this again?

Reply:
like this?